fbw-common/src/systems/instruments/src/OANC/Oanc.tsx

```diff
@@ -391,19 +438,20 @@ export class Oanc<T extends number> extends DisplayComponent<OancProps<T>> {
         this.loadAirportMap(airport);
       });
 
-    this.oansNotAvailable.sub((na) => {
-      if (this.props.messageScreenRef.getOrDefault()) {
-        if (na) {
-          this.props.messageScreenRef.instance.style.visibility = 'visible';
-          this.props.messageScreenRef.instance.innerText = 'NOT AVAIL';
-          this.props.messageScreenRef.instance.classList.add('amber');
-        } else if (this.props.messageScreenRef.instance.innerText === 'NOT AVAIL') {
-          this.props.messageScreenRef.instance.style.visibility = 'hidden';
-          this.props.messageScreenRef.instance.innerText = '';
-          this.props.messageScreenRef.instance.classList.remove('amber');
-        }
-      }
-    }, true);
+    this.sub
+      .on('latitude')
+      .whenChanged()
+      .handle((v) => this.pposLatWord.setWord(v));
```

Comment:
You could also replace these these subscriptions by using `Arinc429LocalVarConsumerSubject` , e.g. something like `private readonly pposLatWord = Arinc429LocalVarConsumerSubject.create(this.sub.on('latitude'));`

Reply:
Also, .whenChanged should be obsolete ;)